README.rst

```diff
@@ -17,7 +19,110 @@ for compatibility purposes.
 
 * ~=v2.0.2: Python ~=2.7,~=3.6 | Django >=1.8,<2.1
 * ~=v2.1: Python ~=2.7,>=3.6,<3.8 | Django >=1.11,<2.1
-* ~=v3.0: __Python >=3.6,<3.9 | Django >=2.0,<2.1 (future release)__
+* ~=v3.0: Python >=3.6,<3.9 | Django 2.2,3.1,3.2 (current release)
+
+Local Development & Testing
+===========================
+
+The following steps should only need to be done once when you first begin:
+
+Install ``pyenv``
+-----------------
+
+These instructions assume that you have `Homebrew <https://brew.sh/>`_ installed,
+but not ``pyenv``.
+
+.. code:: bash
+
+    brew install pyenv
+    touch ~/.bash_profile
+
+Add the following line to your ``~/bash_profile`` or ``.zshrc``::
+
+    eval "$(pyenv init -)"
+
+Reload your shell:
+
+.. code:: bash
+
+    . ~/.bash_profile
+or
+
+.. code:: bash
+
+    . ~/.zshrc
+
+Python Repository Setup
+-----------------------
+
+First, clone the repository and prep your Python environment:
+
+.. code:: bash
+
+    git clone https://github.com/theatlantic/django-select2-forms.git
+    pyenv install 3.6.8
+    pyenv install 3.7.2
+    pyenv install 3.8.0
```

Comment:
Not sure how to know which exactly python versions to install for `3.8` and `3.9`

Reply:
You can check with `pyenv install -l | grep " 3\.[89]\.\d+$"`